sema/check/sinks/yml_sink.py

```
@@ -0,0 +1,10 @@
+# your_submodule/sinks/yml_sink.py
+
+import yaml
+from typing import List
+from ..testing.base import TestResult
+
+
+def write_yml(results: List[TestResult], output_file: str):
```

Ruff

Missing return type annotation for public function `write_yml`

Add return type annotation: `None`

(ANN201)